lib/galaxy/config/config_manage.py

```diff
@@ -516,7 +516,7 @@ def _validate(args, app_desc):
     fp = tempfile.NamedTemporaryFile('w', delete=False, suffix=".yml")
 
     def _clean(p, k, v):
-        return k != 'reloadable'
+        return k not in ['reloadable', 'path_resolves_to']
```

Comment:
Nice! That removes a huge load (I was considering different ways to solve this issue with pykwalify)